components/AnimatedImage/library/src/desktopMain/kotlin/org/jetbrains/compose/animatedimage/DesktopAnimatedImage.kt

```diff
@@ -0,0 +1,86 @@
+package org.jetbrains.compose.animatedimage
+
+import androidx.compose.animation.core.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asComposeImageBitmap
+import org.jetbrains.skia.AnimationFrameInfo
+import org.jetbrains.skia.Bitmap
+import org.jetbrains.skia.Codec
+import java.net.MalformedURLException
+import java.net.URL
+
+private const val DEFAULT_FRAME_DURATION = 50
```

Comment:
I believe browsers usually default to 10 FPS for GIFs with no delay specified. Would it be worth changing this to 100 for consistency?